Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/ScrollingViewIntegrationTest.java

```diff
@@ -129,11 +134,11 @@ public void testScrollingViewCursorTool() {
 		float surfaceWidth = perspective.surfaceWidth;
 		float surfaceHeight = perspective.surfaceHeight;
 
-		int statusBarHeight = 0;
-		int resourceId = mainActivity.getResources().getIdentifier("status_bar_height", "dimen", "android");
-		if (resourceId > 0) {
-			statusBarHeight = mainActivity.getResources().getDimensionPixelSize(resourceId);
-		}
+//		int statusBarHeight = 0;
+//		int resourceId = mainActivity.getResources().getIdentifier("status_bar_height", "dimen", "android");
+//		if (resourceId > 0) {
+//			statusBarHeight = mainActivity.getResources().getDimensionPixelSize(resourceId);
+//		}
```

Comment:
As you already ignored the tests, I would suggest to comment in all lines so the one who does the fixing of the tests knows which lines were in the test case initially.